packages/component-library/stories/MapOverlay.story.js

```diff
@@ -260,6 +280,100 @@ export default () =>
       },
       { notes }
     )
+    .add(
+      "Example: Choropleth + Circle Map",
+      () => {
+        const opacity = number(
+          "Opacity:",
+          0.9,
+          opacityOptions,
+          GROUP_IDS.MARKER
+        );
+
+        const colorOptions = {
+          Thermal: "Thermal",
+          Planet: "Planet",
+          Space: "Space",
+          Earth: "Earth",
+          Ocean: "Ocean"
+        };
+        const colorScheme = select(
+          "Fill Color - Polygons:",
+          colorOptions,
+          colorOptions.Ocean,
+          GROUP_IDS.MARKER
+        );
+        const circleFillColor = object(
+          "Fill Color - Circles:",
+          [255, 182, 193, 200],
+          GROUP_IDS.MARKER
+        );
+        const radiusScaleKnob = number(
+          "Radius Scale:",
+          50,
+          {},
+          GROUP_IDS.MARKER
+        );
+        const radiusScale = radiusScaleKnob || 1;
+
+        const onLayerClick = info => action("Layer Clicked:")(info);
+        const fetchURL = text("Data API URL:", CIVIC_API_URL, GROUP_IDS.DATA);
+
+        return (
+          <DemoJSONLoader urls={[fetchURL]}>
+            {data => {
+              const featuresArrayPath = text(
+                "Features Array Path:",
+                "results.features",
+                GROUP_IDS.DATA
+              );
+              const featuresData = at(data, featuresArrayPath)[0];
+
+              const polygonFieldName = text(
+                "Field Name - Polygons:",
+                "casualtiestotal_day",
+                GROUP_IDS.DATA
+              );
+
+              const circleFieldName = text(
+                "Field Name - Circles:",
+                "casualtiestotal_night",
+                GROUP_IDS.DATA
+              );
+
+              const polygonMinMax = extent(featuresData, f =>
+                parseFloat(f.properties[polygonFieldName])
+              );
+              const polygonColorScale = scaleQuantize()
+                .domain(polygonMinMax)
+                .range(colorSchemeOptions[colorScheme]);
+
+              return (
+                <BaseMap initialZoom={9.99} updateViewport={false}>
+                  <MapOverlay
+                    data={featuresData}
+                    opacity={opacity}
+                    getFillColor={f => {
+                      return f.geometry.type === "Point"
+                        ? circleFillColor
+                        : f.geometry.type === "Polygon" ||
+                          f.geometry.type === "MultiPolygon"
+                        ? polygonColorScale(f.properties[polygonFieldName])
+                        : [0, 0, 0];
+                    }}
+                    radiusScale={radiusScale}
+                    circleFieldName={circleFieldName}
+                    autoHighlight
```

Comment:
`autoHighlight` feels a bit funky without a tooltip, so might remove this line. Makes me curious to see a tooltip implementation, though!